src/engine/x86-64/X86_64Interpreter.v3

```diff
@@ -2202,8 +2206,46 @@ class X86_64InterpreterGen(ic: X86_64InterpreterCode, w: DataWriter) {
 		if (valuerep.tagged) genTagUpdate(BpTypeCode.V128.code);
 		endHandler();
 	}
+	def genStoreLane(opcode: Opcode, size: MemargSize) {
```

Comment:
I think it's shorter to keep using the pattern of passing in the load/store methods.